Gems/EMotionFX/Code/Source/Editor/Plugins/SkeletonOutliner/SkeletonOutlinerPlugin.cpp

```diff
@@ -66,6 +71,8 @@ namespace EMotionFX
     bool SkeletonOutlinerPlugin::Init()
     {
         m_mainWidget = new QWidget(m_dock);
+        [[maybe_unused]] auto* toastNotificationsView =
```

Comment:
This is not needed, right?

Reply:
This was my suggestion to keep just because it can be useful to keep this value around to debug it if needed later